Makefile

```diff
@@ -18,8 +24,6 @@ build-hopr-admin: ## build hopr admin React frontend
 
 .PHONY: build-solidity-types
 build-solidity-types: ## generate Solidity typings
-build-solidity-types: build-cargo
-	npx tsc -p packages/utils/tsconfig.json
 	yarn workspace @hoprnet/hopr-ethereum run build:sol:types
```

Comment:
yarn knows where to find `typechain` and `hardhat`. This is why I didn't remove it there.

On the other end, where `yarn workspaces foreach` triggered sub-`make` invocations, this caused inconvenience with make and can be easily replaced because `yarn` just call `make` under the hood.